packages/create/src/index.ts

```diff
@@ -85,12 +86,21 @@ export async function createRelayNode(
   }
 
   const libp2p = await defaultLibp2p(
-    wakuRelay(options),
+    wakuPubSub(options),
     libp2pOptions,
     options?.userAgent
   );
 
-  return new WakuNode(options ?? {}, libp2p) as RelayNode;
+  const relay = wakuRelay(options);
+
+  return new WakuNode(
+    options ?? {},
+    libp2p,
+    undefined,
```

Comment:
Idea for following improvement: make builders be props of `protocolOptions` object.